bin/problem.py

```diff
@@ -818,15 +816,16 @@ def _validators(
             return problem._validators_cache[key]
 
         assert hasattr(cls, "source_dirs")
-        # TODO #424: We should not support multiple output validators inside output_validator/.
-        paths = [p for source_dir in cls.source_dirs for p in glob(problem.path / source_dir, "*")]
-
-        # Handle default output validation
         if cls == validate.OutputValidator:
-            if not paths:
-                if problem.custom_output:
-                    fatal("Problem validation type requires output_validators/")
+            assert len(cls.source_dirs) == 1
```

Comment:
Is this assertion needed? It's a class-constant, and should not change. And we may want to make it a single string, rather than a list :slightly_smiling_face: